### PR TITLE
Footer: Add Linkedin to social list & remove email us

### PR DESCRIPTION
**Pull request by clmedders**
## Summary

This PR will add LinkedIn to the social media list and remove the email us

Before:
Screenshot 2023-10-03 at 10 45 59 AM

After:
Screenshot 2023-10-03 at 10 52 53 AM

### Preview

[Link to Preview](https://federalist-466b7d92-5da1-4208-974f-d61fd4348571.sites.pages.cloud.gov/preview/gsa/digitalgov.gov/cm-add-linkedin/)

### How To Test

1. pull down branch 
2. run `npm run start`
3. visit `localhost:1313` and scroll to the bottom and verify that the linkedin icon is present and takes you to the digital.gov linkedin page and that the email us has been removed (see below for full requirements)

- The Social media list needs a new item
- LinkedIn icon
- that links to: https://www.linkedin.com/company/digitalgov-gsa/

---

### Dev Checklist

- [x] PR has correct labels
- [x] A11y testing (voice over testing, meets WCAG, run axe tools)
- [x] Code is formatted properly

### CODE REVIEW

File: content/resources/21st-century-integrated-digital-experience-act.md
Comment (mejiaj) [inline content/resources/21st-century-integrated-digital-experience-act.md:1]: This file is showing because it was modified by Prettier.

**Review by mejiaj — DISMISSED** [review]
Thanks for updating this, confirmed links work.

**Review by nick-mon1 — DISMISSED** [review]
LGTM, thanks @clmedders!

**Comment by ToniBonittoGSA** [comment]
@clmedders Please fix the spelling for `LinkedIn` and I'll merge.

**Pull request merged**